app/views/postal_addresses/_postal_address.html.slim

```diff
@@ -0,0 +1,3 @@
+= @user.postal_address.line1.titlecase + " " + @user.postal_address.line2.titlecase + " "
```

Comment:
Hm, this being a bit _contra_ Rails' conventions, I'd say this is making the maintainability of the app a bit harder 😉 

The partial being in `postal_address/_postal_address`, you could render it by just calling
```ruby
= render @user.postal_address
```

Internally, this will pass a local assigned variable with the name `postal_address` to the partial - so there's a variable called `postal_address` available in the partial (same name as the model).

To fully make sure everything is according to plan, you could even `fetch` this variable:
```ruby
# I personally prefer to do a ruby: section in the beginning
ruby:
  postal_address = local_assigns.fetch(:postal_address)

= postal_address.line1

# etc.
```

Using `@user` here would be bypassing all this nice assignment magic that Rails does for us. Also - and that's the main point - it makes the partial incompatible for any other use aside from views with an `@user` set 😢